accessibility-checker.php

```diff
@@ -129,6 +129,30 @@
 require_once plugin_dir_path( __FILE__ ) . 'includes/purge.php';
 require_once plugin_dir_path( __FILE__ ) . 'includes/system-info.php';
 
+/**
+ * Include Rules
+ */
+require_once plugin_dir_path( __FILE__ ) . 'includes/classes/class-rules.php';
+require_once plugin_dir_path( __FILE__ ) . 'includes/classes/rules/class-rule.php';
+require_once plugin_dir_path( __FILE__ ) . 'includes/classes/rules/class-rule-aria-hidden.php';
+( new EDAC\Rules() )->add_rule( 'aria_hidden', new EDAC\Rules\Rule_Aria_Hidden() );
+add_filter(
+	'edac_filter_register_rules',
+	function ( $rules ) {
+		$rules_from_objects = ( new EDAC\Rules() )->get_rules();
+		foreach ( $rules_from_objects as $rule ) {
+			$rules[] = array(
+				'title'     => $rule->get_title(),
+				'info_url'  => $rule->get_info_url(),
+				'slug'      => $rule->get_slug(),
+				'rule_type' => $rule->get_type(),
+				'summary'   => $rule->get_summary(),
+			);
+		}
+		return $rules;
+	}
+);
+
```

Comment:
This is just for the POC, to show how we can add rules in a manner that is backwards-compatible.